crates/smelt-data/client.data.proto

```diff
@@ -68,9 +68,15 @@ message Ulimit {
   optional int64 hard = 3;
 }
 
+enum RunMode {
```

Comment:
this should be included in the CfgDocker message, state local to the docker executor

Reply:
oh, just read your hardcoded comment, makes sense